coffee-se/coffee-se-api/src/main/java/hu/icellmobilsoft/coffee/se/api/exception/enums/CoffeeFaultType.java

```diff
@@ -31,4 +31,8 @@ public enum CoffeeFaultType {
      * DTO conversion fail
      */
     DTO_CONVERSION_FAILED,
+    /**
+     * Operation failed
+     */
+    OPERATION_FAILED,
```

Comment:
Ha ugyan azt a nevet használjuk itt mint ami a hu.icellmobilsoft.coffee.dto.exception.enums.CoffeeFaultType-ba van az nem biztos hogy jó lesz. Nem lehetne inkább valami specifikusabb mint a DTO_CONVERSION_FAILED? Pl JSON_CONVERSION_FAILED ?

Reply:
azért lett ez a neve, mert jelenleg is ez a faultType van használva gson esetén, ha ettől most eltérünk, akkor ez majd a clienseket is érintheti (vagy projekteken ezt külön le kell kezelni), így célszerűbbnek tünt az OPERATION_FAILED-re rakni